addons/binding/org.openhab.binding.sleepiq/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,7 @@
+eclipse.preferences.version=1
```

Comment:
The fact that some are missing do cause warnings on the maven build:

```
[12:56:05][Step 2/4] [INFO] --- tycho-compiler-plugin:1.0.0:compile (default-compile) @ org.openhab.binding.allplay ---
[12:56:05][Step 2/4] [WARNING] Parameter 'useProjectSettings' is set to true, but preferences file '/opt/buildagent/work/5c3cd640ded15833/addons/binding/org.openhab.binding.allplay/.settings/org.eclipse.jdt.core.prefs' could not be found!
[12:56:05][Step 2/4] [INFO] Using Groovy-Eclipse compiler to compile both Java and Groovy files
```

This error is reported at least 12 times